melodic/distribution.yaml

```diff
@@ -5048,6 +5048,17 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: master
     status: developed
+  marvelmind_nav:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/MarvelmindRobotics/marvelmind_nav-release.git
+      version: 1.0.10-1
+    source:
+      test_pull_requests: false
```

Comment:
We can only support pull request builds on github due to using https://plugins.jenkins.io/ghprb/

Reply:
I just added license headers to all source files in the source repository. Do I need to create new release and/or pull request? Thank you

Reply:
Yes, that's the easiest thing to do.  Bloom will automatically reopen the PR for you.